sync/src/relayer/mod.rs

```diff
@@ -173,12 +174,17 @@ impl Relayer {
         message: packed::RelayMessageUnionReader<'r>,
     ) {
         let item_name = message.item_name();
+        let item_bytes = message.as_slice().len() as u64;
         let status = self.try_process(Arc::clone(&nc), peer, message);
 
-        metrics!(counter, "ckb-net.received", 1, "action" => "relay", "item" => item_name.to_owned());
-        if !status.is_ok() {
-            metrics!(counter, "ckb-net.status", 1, "action" => "relay", "status" => status.tag());
-        }
+        metrics!(
```

Comment:
I think this metrics should be moved to https://github.com/nervosnetwork/ckb/blob/a8829f4a54ec453c48b67808c5ec694a09cfae2a/sync/src/synchronizer/mod.rs#L673 

and use the same metrics parameters as direction out
```
    metrics!(
        counter,
        "ckb.messages_bytes", bytes,
        "direction" => "in",
        "protocol_id" => SupportProtocols::Sync::protocol_id(),
        "item_id" => msg.item_id()
    );
```

Reply:
My fault.
Updated, thanks a lot.